src/screens/TestCases/ReviewCommentMaster/ReviewCommentMasterComponent.js

```diff
@@ -0,0 +1,218 @@
+import React, { useEffect, useState } from "react";
+import DataTable from "react-data-table-component";
+import PageHeader from "../../../components/Common/PageHeader";
+import { ExportToExcel } from "../../../components/Utilities/Table/ExportToExcel";
+import ReviewCommentMasterModal from "./Validation/ReviewCommentMasterModal";
+import { useDispatch, useSelector } from "react-redux";
+import { getReviewCommentMasterListThunk } from "../../../redux/services/testCases/reviewCommentMaster";
+import { customSearchHandler } from "../../../utils/customFunction";
+import TableLoadingSkelton from "../../../components/custom/loader/TableLoadingSkelton";
+import { Col, Row } from "react-bootstrap";
+
+function ReviewCommentMasterComponent() {
+  const dispatch = useDispatch();
+
+  // // redux state
+  const { reviewCommentMasetrList, isLoading } = useSelector(
```

Comment:
Add spell check extention,
1.reviewCommentMasetrList
2.filteredRevieCommentMasterList

Reply:
Added spell check extention, and updated changes